ts/src/pro/coinex.ts

```diff
@@ -1030,13 +1079,21 @@ export default class coinex extends coinexRest {
     }
 
     handleAuthenticationMessage (client: Client, message) {
+        //
+        // success
         //
         //     {
-        //         "error": null,
-        //         "result": {
-        //             "status": "success"
-        //         },
-        //         "id": 1
+        //         "id": 1,
+        //         "code": 0,
+        //         "message": "OK"
+        //     }
+        //
+        // fail
+        //
+        //     {
+        //         "id": 1,
+        //         "code": 21002,
+        //         "message": ""
         //     }
         //
         const messageHashSpot = 'authenticated:spot';
```

Comment:
if the user does `asyncio.gather(ex.watch_orders('BTC/USDT'), ex.watc_orders('BTC/USDT:USDT'))`  it will trigger "concurrent" authenticate() calls, so I think this might be a problem, blindly resolve both of them